compare-master-to-stable.js

```diff
@@ -85,44 +85,43 @@ var checkout = oneArg(exec('git checkout %s'));
 
 var getCurrentBranch = andThen(noArgs(exec('git rev-parse --abbrev-ref HEAD')), oneLine);
 var getTags = noArgs(exec('git tag'));
-var getShaOfTag = oneArg(exec('git rev-list %s | head -n 1'));
 var getTheLog = oneArg(exec('git log --pretty=oneline %s..HEAD | cat'));
 
 // remember this so we can restore state
 var currentBranch;
 
 getCurrentBranch().
-then(function (branch) {
+then(function(branch) {
   currentBranch = branch;
 }).
 then(getTags).
-then(function (tags) {
+then(function(tags) {
   return tags.
     filter(semver.valid).
     map(semver.clean).
     sort(semver.rcompare);
 }).
-then(function (tags) {
+then(function(tags) {
   var major = tags[0].split('.')[0];
   return tags.
-    filter(function (ver) {
-      return semver(ver).major == major;
+    filter(function(ver) {
+      return semver(ver).major === major;
```

Comment:
Afaict, `major` will be a string and `semver(ver).major` will be a number, so `===` won't work.
I think it's fine to convert `major` to a number (since it is only used locally for comparison against `semver(ver).major`).